packages/contracts/README.md

```diff
@@ -251,15 +251,22 @@ Next, you should uncomment the following lines in `foundry.toml`.
 # via-ir = true 
 ```
 
-And then you should uncomment the commented imports and functions in `src/utils/ZKSyncCreate2Factory.sol`
+Uncomment the following commented-out files. 
+These are commented out in the files for ZkSync to avoid problems when testing in foundry.
```

Comment:
Is it possible to avoid commenting out those zksync files by adding an option to the command of `yarn build` that excludes zksync files?

Reply:
@SoraSuegami Maybe we can use --skip flag, let me try it.